.gitlab-ci.yml

```diff
@@ -280,6 +293,17 @@ before_script:
   UPGRADE_TEST: "graceful"
   STARTUP_SCRIPT: ""
 
+.ubuntu_canal_kubeadm_variables: &ubuntu_canal_kubeadm_variables
+# stage: deploy-gce-part1
+  KUBE_NETWORK_PLUGIN: canal
+  AUTHORIZATION_MODES: "{ 'authorization_modes':  [ 'RBAC' ] }"
```

Comment:
Note: RBAC and the Node authorizer are always on.

Reply:
There is internal logic to determine if rbac is enabled. We will refactor this soon.